doc/api/url.md

```diff
@@ -11,28 +11,27 @@ const url = require('url');
 
 ## URL Strings and URL Objects
 
-A URL String is a structured string containing multiple meaningful components. 
-When parsed, a URL Object is returned containing properties for each of these
+A URL string is a structured string containing multiple meaningful components. 
+When parsed, a URL object is returned containing properties for each of these
 components.
 
 The following details each of the components of a parsed URL. The example
 `'http://user:pass@host.com:8080/p/a/t/h?query=string#hash'` is used to
 illustrate each.
 
 ```
-+-----------------------------------------------------------------------------+
-| href                                                                        |
-+----------+--+-----------+------------------+-------------------------+------+
-| protocol |* | auth      | host             | path                    | hash |
-|          |  |           +-----------+------+----------+--------------+      |
-|          |  |           | hostname  | port | pathname | search       |      |
-|          |  |           |           |      |          +-+------------+      |
-|          |  |           |           |      |          | | query      |      |
-" http:     //  user:pass @  host.com : 8080 / p/a/t/h  ? query=string #hash  "
-|          |  |           |           |      |          | |            |      |
-+----------+--+-----------+-----------+------+----------+-+------------+------+
++---------------------------------------------------------------------------+
+| href                                                                      |
++----------++-----------+-----------------+-------------------------+-------+
+| protocol || auth      | host            | path                    | hash  |
+|          ||           +----------+------+----------+--------------+       |
+|          ||           | hostname | port | pathname | search       |       |
+|          ||           |          |      |          +-+------------+       |
+|          ||           |          |      |          | | query      |       |
+" http:    // user:pass @ host.com : 8080 | /p/a/t/h |?query=string | #hash "
```

Comment:
I think the `|` between the `port` and `pathname` should just be removed, same with the `|` between `pathname` and `search`, and with the `|` between `search` and `hash`.